src/mscorlib/corefx/System/Globalization/TextInfo.cs

```diff
@@ -38,38 +36,28 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         ////
 
         [OptionalField(VersionAdded = 2)]
-        private String m_listSeparator;
```

Comment:
@stephentoub thanks for updating me. I think we need to revisit this decision as I believe we'll run into scenarios with the cross-plat support. I know serialization is always headache. people already started to log issues against it:

https://github.com/dotnet/corefx/issues/10513